confd/templates/floodlight/floodlightkilda.properties.tmpl

```diff
@@ -80,3 +80,5 @@ org.openkilda.floodlight.switchmanager.SwitchManager.connect-mode=AUTO
 org.openkilda.floodlight.switchmanager.SwitchManager.broadcast-rate-limit=200
 org.openkilda.floodlight.switchmanager.SwitchManager.unicast-rate-limit=200
 org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size=250
+org.openkilda.floodlight.switchmanager.SwitchManager.flow-meter-burst-coefficient=0.1
+org.openkilda.floodlight.switchmanager.SwitchManager.system-rule-meter-burst-coefficient=0.1
```

Comment:
`-rule` is excess piece in this name. To make both names alike, is should be `system-meter-burst-coefficient`.

(minor)